cmd/minikube/cmd/cache_list.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2017 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"text/template"
+
+	"github.com/spf13/cobra"
+	cmdConfig "k8s.io/minikube/cmd/minikube/cmd/config"
+	"k8s.io/minikube/pkg/minikube/constants"
+)
+
+const cacheListFormat = "- {{.CacheImageName}}\n"
```

Comment:
Nit: do we need the "-" character here? It would make automatic parsing of stdout a little tricker (like if you wanted to use this in a bash for loop, etc.).

Reply:
Yes, I ignored this usage scenario, I'll remove this character to bring about this results:
```sh
$ minikube cache add redis ubuntu
$ minikube cache delete $(minikube cache list)
```